README.md

```diff
@@ -372,6 +372,37 @@ require("aerial").setup({
   -- Run this command after jumping to a symbol (false will disable)
   post_jump_cmd = "normal! zz",
 
+  -- Invoked after each symbol is parsed, can be used to modify the parsed item,
+  -- or to filter it by returning false.
+  --
+  -- bufnr: a neovim buffer number
+  -- item: of type aerial.Symbol
+  -- ctx: a record containing the following fields:
+  --   * backend_name: treesitter, lsp, man...
+  --   * lang: info about the language
+  --   * symbols?: specific to the lsp backend
+  --   * symbol?: specific to the lsp backend
+  --   * syntax_tree?: specific to the treesitter backend
+  --   * match?: specific to the treesitter backend, TS query match
+  post_parse_symbol = function(bufnr, item, ctx)
+    return true
+  end,
+
+  -- Invoked after all symbols have been parsed and post-processed,
+  -- allows to modify the symbol structure before final display
+  --
+  -- bufnr: a neovim buffer number
+  -- items: a collection of aerial.Symbol items, organized in a tree,
+  --        with 'parent' and 'children' fields
+  -- ctx: a record containing the following fields:
+  --   * backend_name: treesitter, lsp, man...
+  --   * lang: info about the language
+  --   * symbols?: specific to the lsp backend
+  --   * syntax_tree?: specific to the treesitter backend
+  post_add_all_symbols = function(bufnr, items, ctx)
+    return items
+  end,
+
```

Comment:
This section is actually auto-generated, so you'll have to add it to `config.lua` and it will get copied here.

Reply:
How does it get copied, which command should I run? Because presumably I must run the script before committing, right?

Reply:
It actually [gets run in CI](https://github.com/stevearc/aerial.nvim/blob/master/.github/workflows/update-docs.yml), but if you want to run it locally you can do `python .github/main.py generate`. Note that you will probably have to `pip install pyparsing==3.0.9` before, and you may want to do both of those from inside a virtualenv (this is why I have a python layout in the [.envrc](https://github.com/stevearc/aerial.nvim/blob/master/.envrc))